02-peer-protocol.md

```diff
@@ -2233,6 +2233,7 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
     1. type: 0 (`blinded_path`)
     2. data:
         * [`point`:`path_key`]
+    1. type: 1 (`accountable`)
```

Comment:
Why not allow multiple accountability level instead of it being just a boolean flag? Just as with the endorsement before.